wiki/entry_engel.qmd

```diff
@@ -0,0 +1,33 @@
+ 
+Berryman, C., Ferguson, C. J., & Negy, C. (2018). Social media use and mental health among young adults. Psychiatric quarterly, 89, 307-314.
+ 
+Primack, B. A., Shensa, A., Escobar-Viera, C. G., Barrett, E. L., Sidani, J. E., Colditz, J. B., & James, A. E. (2017). Use of multiple social media platforms and symptoms of depression and anxiety: A nationally-representative study among US young adults. Computers in human behavior, 69, 1-9.
+ 
+Lin, L. Y., Sidani, J. E., Shensa, A., Radovic, A., Miller, E., Colditz, J. B., ... & Primack, B. A. (2016). Association between social media use and depression among US young adults. Depression and anxiety, 33(4), 323-331.
+
+Die Studie von Berryman, Feruson und Negy (2018) untersucht das Verhältnis zwischen der Nutzung digitaler Medien und der psychischen Gesundheit bei jungen Menschen. 
+Ausgangspunkt für die Untersuchung der Studie ist die Umstrittenheit des Zusammenhangs zwischen der Social Media Nutzung und dessen Einfluss auf das psychische Wohlbefinden.
+Dafür wurden 467 Studenten mit einem Durchschnittsalter von 20 Jahren, die eine Universität im Südosten der Vereinigten Staaten besuchten, befragt. Die Befragung erfolgte im Hinblick auf die Zeit, die sie mit der Nutzung der sozialen Medien verbringen, die Bedeutung sozialer Medien in ihrem Leben und die Tendenz zum „Vaguebooking“. „Vaguebooking“, stellt hierbei das Posten von unklaren, aber alarmierenden Nachrichten dar, die Aufmerksamkeit erregen sollen. Die untersuchten Ergebnisse beinhalten allgemeine psychologische Gesundheitssymptome, Selbstmordgedanken, Einsamkeit, soziale Ängste und verminderte Empathie.
+Die Messung erfolgte auf einer Likert-Skala. Zu „Vaguebooking" gab es drei gezielte Fragen. Zeitaufwand für social Media und der Umgang mit sozialen Medien wurde mit einer Einzelfrage gemessen. Die soziale Unterstützung wurde beispielsweise über eine Multidimensionale Skala erfasst. Neben anderen Prädiktoren war die soziale Erwünschtheit mit einer geringen Berichterstattung über negative Symptome verbunden. Soziale Unterstützung war ein konsistenter Schutzfaktor für alle negativen Ereignisse wie auch Suizidgedanken. Wahrgenommene Eltern-Kind-Konflikte wurden mit psychischen Gesundheitssympotomen, Selbstmord und Einsamkeit in Verbindung gebracht.
+Es konnte dargelegt werden, dass Social Media ein schlechter Prädiktor für psychische Krankheiten darstellt und somit die Nutzung sozialer Medien keine Vorhersage dafür ist, dass die psychische Gesundheit beeinträchtigt wird. „Vaguebooking“ war jedoch vorhersagend für Selbstmordgedanken. Die Ergebnisse deuten insgesamt daraufhin, mit Ausnahme von Vaguebooking, dass die Nutzung von digitalen Meiden keinen negativen Einfluss auf die psychische Gesundheit hat.
+Zusätzlich konnte gezeigt werden das der Eltern-Kind-Konflikt eine der stärksten Prädiktoren für psychische Probleme ist. Es konnte bewiesen werden, dass es keine Beweise dafür gibt das digitale Medien psychische Krankheuten begünstigen.
+
```

Comment:
The study by Berryman, Feruson, and Negy (2018) examines the relationship between digital media use and mental health among young people. 
The starting point for the study's investigation is the controversial nature of the relationship between social media use and its impact on mental well-being.
For this purpose, 467 students with an average age of 20 attending a university in the southeastern United States were surveyed. The survey was conducted with regard to the amount of time they spend using social media, the importance of social media in their lives, and the tendency to "vaguebooking." "Vaguebooking", here represents the posting of unclear but alarming messages intended to attract attention. Outcomes studied included general psychological health symptoms, suicidal thoughts, loneliness, social anxiety, and decreased empathy.
Measurement was on a Likert scale. There were three targeted questions on "vaguebooking." Time spent on social media and social media use were measured with a single question. Social support, for example, was measured using a multidimensional scale. Among other predictors, social desirability was associated with low reporting of negative symptoms. Social support was a consistent protective factor for all negative events as well as suicidal ideation. Perceived parent-child conflict was associated with mental health symptoms, suicide, and loneliness.
It was demonstrated that social media is a poor predictor of mental illness and thus social media use is not predictive of impaired mental health. "Vaguebooking," however, was predictive of suicidal thoughts. Overall, the results suggest, with the exception of vaguebooking, that digital avoidance use does not negatively impact mental health.
Additionally, parent-child conflict was shown to be one of the strongest predictors of mental health problems. It could be proven that there is no evidence that digital media promote mental illness.